### PR TITLE
remove hover effect on checkbox

### LINKED ISSUE #2165: Hover-Effect: Remove checkmark and square icon on checkbox on hover
1.
image

Remove the square on hover.

2.
image

Remove the checkmark on hover.

3.
image

Remove the checkmark on hover.

### PR DESCRIPTION
**Pull request by marcwelz**
Fixes #2165

# Merge Checklist:
- [x] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch-webhub-cloud/pattern-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [x] Tests are sufficient.
- [x] Modifications still work in IE.
- [x] Documentation is up to date.
- [x] Changelog is up to date.
- [x] Typescript typings for properties are up to date.
- [x] Designers approved changes or no approval needed.
- [x] Dependencies to other components still work.

### CODE REVIEW

File: src/components/10-atoms/checkbox/index.scss [hunk 1]
```diff
@@ -127,90 +106,62 @@ axa-checkbox {
...
       }
     }
-
     .a-checkbox__input:checked + .a-checkbox__icon::after {
       border-color: $color-silver;
       background-color: $color-silver;
     }
   }
-
   &[error] {
```
Comment (raphaellueckl) [inline src/components/10-atoms/checkbox/index.scss:165]: Why did you remove all line breaks here?

**Comment by raphaellueckl** [comment]
I noticed, that "checkmark-inverted" has no hover effect now. But fine for me.

File: src/components/10-atoms/checkbox/index.scss (hunk 1 above)
Comment (marcwelz) [reply src/components/10-atoms/checkbox/index.scss:165]: idk, was prettier probably but I changed it back

**Review by raphaellueckl — APPROVED** [review]
